Automattic/wp-calypso
# Stepper Flow: Blogger/Builder Flow and the first two steps of Sell

### Changes proposed in this Pull Request

Using the new stepper framework, we recreated the **builder** and the **blogger** flow, starting from the **intent** gathering step.

### Testing instructions




* Checkout the branch
* Run `yarn start`
* Go to `http://calypso.localhost:3000/stepper`
* Check that the builder flow is working



### What has been done

- Signup Steps: 
    - Site setup flow ( _client/landing/stepper/declarative-flow/site-setup-flow.ts_ ): This flow starts from the intent gathering screen and let you go through the blogger or the builder flows.
    - The following steps from signup have been added:
        - _courses_
        - _design-setup_
        - _site-options_
        - _starting-point_, now _blogger-starting-point_ 
        - _store-features_
    - store-features ( _client/signup/steps/store-features_ ): intents have been extracted to their own file ( _client/signup/steps/store-features/intents.tsx_ ), so to be able to import them from the stepper store-feature
- Onboard store ( _packages/data-stores/src/onboard_ ): added actions/selectors/reducers for `intent`, `startingPoint` and `storeType`
- Site store ( _packages/data-stores/src/site_ ): Added site tagline
- Web Preview component ( _client/components/web-preview/component.jsx_ ): Exported the connected component version to its own file, so to be able to load the connected and not-connected version.
- Onboarding components ( _packages/onboarding_ ):  Simplified the StepContainer component, added CSS classes and moved out the CSS regarding a single step
